src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return Arrays.stream(numbers.stream().map(s -> s.split(COMA))
+                .flatMapToInt(arr -> Arrays.stream(arr).mapToInt(Integer::parseInt))
+                .toArray())
```

Comment:
Looks pretty complicated, let's simplify it.
```suggestion
        return numbers.stream()
                .flatMap(s -> Arrays.stream(s.split(COMA)))
                .mapToInt(Integer::parseInt)
```